doc/_sphinx/theme/layout.html

```diff
@@ -124,6 +124,11 @@
 
 <div class="main-area">
   <div class="document-wrapper">
+    <div class="warning hidden" id="version-warning">
+      <p><strong>Warning:</strong> you are currently viewing the docs for a specific
+      version <span class="version"></span> of Flame.</p>
+      <p>Please <a href="/main">click here</a> to go see the most up-to-date documentation.</p>
```

Comment:
```suggestion
      <p><strong>Warning:</strong> you are currently viewing the docs for an older
      version <span class="version"></span> of Flame.</p>
      <p>Please <a href="/">click here</a> to go see the documentation for the latest
      released version.</p>
```
I suggest that we link to the latest released version instead of `main`, since people might try to use unreleased features if we do.